base/bucket_gocb.go

```diff
@@ -119,6 +119,10 @@ func QueryBucketItemCount(ctx context.Context, n1qlStore N1QLStore) (itemCount i
 	return val.Count, nil
 }
 
+func IsMinimumVersion(major, minor, requiredMajor, requiredMinor uint64) bool {
+	return isMinimumVersion(major, minor, requiredMajor, requiredMinor)
```

Comment:
Was there a reason we added a new exposed method instead of changing the private one to public?